### ISSUE TITLE
OS-wide buffering bug with bt headphones, crashes headphones

### ISSUE THREAD

**Issue by bartgrantham** [madronalabs/vutu#6]
I was using bluetooth headphones, Sony WH-1000XM4's set up as the system default audio output, for my first run of vutu.  When I start Vutu, my OS audio gets worse and worse lag (even in things like Quicklook in Finder) until it's several seconds behind UI events (such as hitting the "play" button in Vutu).  Once it reaches this point my headphones crash and silently turn off.

I assume this is some toxic interaction between the system audio, the audio driver library, and the bluetooth codec.  With wired headphones there's no issue.

**Comment by randyjonesworld** [comment]
Thanks for the report, that's pretty weird.

Because you mention Finder I guess you're on OS X. Please add further details including machine type and OS version.

Vutu uses [rtaudio](https://github.com/thestk/rtaudio) for audio I/O. So it's likely the problem lies there. If you're able to compile a test rtaudio app and try it, you might be able to make a useful report for the rtaudio folks.  If not, I totally understand.

**Comment by bartgrantham** [comment]
Yeah, I've never experienced a phenomenon like this before.  Having the headphones suddenly turn off was surprising!

This is on an M1 Macbook Air, Sonoma 14.4.1.  I don't think I have the bandwidth to dig into rtaudio at the moment, but if I get the chance I'll follow up with them. 

This issue can be closed, IMHO.  If there's another report with a different configuration it can contribute to the bug evidence then.
